theano/gpuarray/dnn_base.c

```diff
@@ -114,6 +126,17 @@ c_set_filter(PyGpuArrayObject *var, cudnnFilterDescriptor_t desc) {
   return 0;
 }
 
+static int c_make_filter(PyGpuArrayObject *var, cudnnFilterDescriptor_t *desc) {
```

Comment:
This can probably be used to clean up code. If that is the case, it would be great to include it. Quick to do and make a big difference in readability.